app/templates/components.html

```diff
@@ -12,10 +12,16 @@
         <div class="dropdown-menu dropdown-menu-right">
             {% set list1 = g.user['email'].split('@') %}
             {% set userid = list1[0] %}
+            {% set curr_user = user%}
             <a class="dropdown-item" href="{{ url_for('search') }}">Search</a>
             <a class="dropdown-item" href="{{ url_for('portfolio', ucinet=userid) }}">Portfolio</a>
             <a class="dropdown-item" href="{{ url_for('settings.settings') }}">Settings</a>
             <a class="dropdown-item" href="{{ url_for('faq.faq') }}">FAQ</a>
+            {% if curr_user != null%}

```

Comment:
Can we investigate if this line can be rephrased into `( (curr_user != None) and (curr_user['is_admin'] == 'True') ) ` ?

Reply:
Got it!